src/mwoffliner.lib.ts

```diff
@@ -420,17 +427,31 @@ async function execute(argv: any) {
 
     logger.log('Getting articles')
     stime = Date.now()
-    const { jsModuleDependencies, cssModuleDependencies } = await saveArticles(zimCreator, downloader, dump, forceRender)
+    const { jsModuleDependencies, cssModuleDependencies, jsMobileModuleDependencies, cssMobileModuleDependencies } = await saveArticles(
+      zimCreator,
+      downloader,
+      dump,
+      hasWikimediaMobileApi,
+      forceRender,
+    )
     logger.log(`Fetching Articles finished in ${(Date.now() - stime) / 1000} seconds`)
 
     logger.log(`Found [${jsModuleDependencies.size}] js module dependencies`)
     logger.log(`Found [${cssModuleDependencies.size}] style module dependencies`)
 
-    const allDependenciesWithType = [
+    let allDependenciesWithType = [
       { type: 'js', moduleList: Array.from(jsModuleDependencies) },
       { type: 'css', moduleList: Array.from(cssModuleDependencies) },
     ]
 
+    if (dump.opts.isMobileRenderer) {
```

Comment:
this should be part of the mobile renderer

Reply:
I decided to remove this condition because I can't correctly check the render type at this stage of the scrapping process.